### PR TITLE
support LaTeX commands for greek letters and Unicode super-/subscript characters in pasted formulas

### PR DESCRIPTION
**Pull request by bsoelch**

### CODE REVIEW

**Review by alexhuntley — CHANGES_REQUESTED** [review]
This looks good, but please could you reuse the Greek letters found in `data.py` rather than duplicating them in this file?

File: plots/parser.py [hunk 1]
```diff
@@ -98,9 +104,18 @@ def prod(self, items):
          | subscriptsuperscript
          | superscriptsubscript
 
+GREEK_LETTER_NAME : "\\Alpha" | "\\Beta" | "\\Gamma" | "\\Delta" | "\\Epsilon" | "\\Zeta"
```
Comment (alexhuntley) [inline plots/parser.py:107]: Could you do the same for this too? I'm not sure what the best method would be, perhaps using an f-string

**Review by alexhuntley — CHANGES_REQUESTED** [review]

File: plots/parser.py (hunk 1 above)
Comment (bsoelch) [reply plots/parser.py]: I think i have solved it.

**Review by alexhuntley — APPROVED** [review]

**Comment by alexhuntley** [comment]
Nice work, thanks very much :)

**Pull request merged**